### PR TITLE
Notebook Tests

### PR DESCRIPTION
**Pull request by AnonymousCodes911**
fixes: #52 
continued: #98 
conformed the `tox.ini` file to linux
fixes to `build_docs`

### CODE REVIEW

**Comment by matteobachetti** [comment]
I meant *literally* to install stingray, update the notebooks submodule to the version of notebooks of the PR, and run the stingray tox test to build the docs. 

You can do all these steps inside the test.

There is no need to copy pieces of the Stingray documentation in the notebooks repository

File: .github/workflows/build_test_notebooks.yml [hunk 1]
```diff
@@ -7,40 +8,45 @@ on:
...
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout stingray repo
         uses: actions/checkout@v4
         with:
           repository: StingraySoftware/stingray
           path: ./stingray
+
       - name: Checkout notebooks repo
```
Comment (matteobachetti) [inline .github/workflows/build_test_notebooks.yml:23]: Is this step needed?

File: .github/workflows/build_test_notebooks.yml [hunk 2]
```diff
@@ -7,40 +8,45 @@ on:
...
-          pip install sphinx sphinx-astropy nbsphinx tox 
+          pip install sphinx sphinx-astropy tox
+
       - name: Install Stingray
-        run: pip install -e git+https://github.com/StingraySoftware/stingray.git#egg=stingray
+        run: pip install -e ./stingray
+
       - name: Update submodules
         run: |
+          cd ./stingray
```
Comment (matteobachetti) [inline .github/workflows/build_test_notebooks.yml:46]: What you are updating here is the notebook submodule version already in stingray. You should add an instruction to checkout the exact version of the notebooks of the PR

**Review by matteobachetti — CHANGES_REQUESTED** [review]

File: .github/workflows/build_test_notebooks.yml (hunk 1 above)
Comment (AnonymousCodes911) [reply .github/workflows/build_test_notebooks.yml]: No, I fixed the branch in the latest commit so this is not required here, thank you!

File: .github/workflows/build_test_notebooks.yml (hunk 2 above)
Comment (AnonymousCodes911) [reply .github/workflows/build_test_notebooks.yml]: fixed in latest commit

**Pull request merged**